css/types/number.json

```diff
@@ -25,22 +25,22 @@
               "version_added": "5"
             },
             "opera": {
-              "version_added": true
+              "version_added": "2"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "10.1"
             },
             "safari": {
               "version_added": "1"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "1"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "1.0"
             },
             "webview_android": {
-              "version_added": true
+              "version_added": "1"
```

Comment:
Did you intend to change this to `"1"`?

Reply:
Yes. My believe is that numbers in CSS are such a basic functionality that this must have been there from the start. What would you say?

Reply:
Leave it as is. I was just making sure.